[2mdn.net] HTTPConnectionPool(host='2mdn.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f28184447d0>: Failed to resolve '2mdn.net' ([Errno -5] No address associated with hostname)"))
[a-msedge.net] HTTPConnectionPool(host='a-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f28184463d0>: Failed to resolve 'a-msedge.net' ([Errno -5] No address associated with hostname)"))
[a2z.com] HTTPConnectionPool(host='a2z.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818444110>: Failed to resolve 'a2z.com' ([Errno -5] No address associated with hostname)"))
[aaplimg.com] HTTPConnectionPool(host='aaplimg.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281844c310>: Failed to resolve 'aaplimg.com' ([Errno -5] No address associated with hostname)"))
[adnxs.com] HTTPConnectionPool(host='adnxs.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281844e010>: Failed to resolve 'adnxs.com' ([Errno -5] No address associated with hostname)"))
[aiv-cdn.net] HTTPConnectionPool(host='aiv-cdn.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818445650>: Failed to resolve 'aiv-cdn.net' ([Errno -5] No address associated with hostname)"))
[akadns.net] HTTPConnectionPool(host='akadns.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818506b10>: Failed to resolve 'akadns.net' ([Errno -5] No address associated with hostname)"))
[akamai.net] HTTPConnectionPool(host='akamai.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f28185142d0>: Failed to resolve 'akamai.net' ([Errno -5] No address associated with hostname)"))
[akamaiedge.net] HTTPConnectionPool(host='akamaiedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281844d6d0>: Failed to resolve 'akamaiedge.net' ([Errno -5] No address associated with hostname)"))
[akamaihd.net] HTTPConnectionPool(host='akamaihd.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f28184f65d0>: Failed to resolve 'akamaihd.net' ([Errno -5] No address associated with hostname)"))
[akamaized.net] HTTPConnectionPool(host='akamaized.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818516310>: Failed to resolve 'akamaized.net' ([Errno -5] No address associated with hostname)"))
[amazon-adsystem.com] HTTPConnectionPool(host='amazon-adsystem.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818517f50>: Failed to resolve 'amazon-adsystem.com' ([Errno -5] No address associated with hostname)"))
[apple-dns.net] HTTPConnectionPool(host='apple-dns.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818517d50>: Failed to resolve 'apple-dns.net' ([Errno -5] No address associated with hostname)"))
[azureedge.net] HTTPConnectionPool(host='azureedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e5883d0>: Failed to resolve 'azureedge.net' ([Errno -5] No address associated with hostname)"))
[azurewebsites.net] HTTPConnectionPool(host='azurewebsites.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e58b9d0>: Failed to resolve 'azurewebsites.net' ([Errno -5] No address associated with hostname)"))
[b-msedge.net] HTTPConnectionPool(host='b-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818529410>: Failed to resolve 'b-msedge.net' ([Errno -5] No address associated with hostname)"))
[bytefcdn-oversea.com] HTTPConnectionPool(host='bytefcdn-oversea.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d982a90>: Failed to resolve 'bytefcdn-oversea.com' ([Errno -5] No address associated with hostname)"))
[casalemedia.com] HTTPConnectionPool(host='casalemedia.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f281d980910>: Failed to establish a new connection: [Errno 111] Connection refused'))
[cdn77.org] HTTPConnectionPool(host='cdn77.org', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d92bfd0>: Failed to resolve 'cdn77.org' ([Errno -5] No address associated with hostname)"))
[cdninstagram.com] HTTPConnectionPool(host='cdninstagram.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d92a1d0>: Failed to resolve 'cdninstagram.com' ([Errno -5] No address associated with hostname)"))
[cloudapp.net] HTTPConnectionPool(host='cloudapp.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d929010>: Failed to resolve 'cloudapp.net' ([Errno -5] No address associated with hostname)"))
[cloudfront.net] HTTPConnectionPool(host='cloudfront.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d9ffe90>: Failed to resolve 'cloudfront.net' ([Errno -5] No address associated with hostname)"))
[demdex.net] HTTPConnectionPool(host='demdex.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281da9a150>: Failed to resolve 'demdex.net' ([Errno -5] No address associated with hostname)"))
[dns.google] HTTPConnectionPool(host='dns.google', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f281e54d3d0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
[domaincontrol.com] HTTPConnectionPool(host='domaincontrol.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f281ddcba90>: Failed to establish a new connection: [Errno 111] Connection refused'))
[edgekey.net] HTTPConnectionPool(host='edgekey.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281dc8e890>: Failed to resolve 'edgekey.net' ([Errno -5] No address associated with hostname)"))
[edgesuite.net] HTTPConnectionPool(host='edgesuite.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281dc845d0>: Failed to resolve 'edgesuite.net' ([Errno -5] No address associated with hostname)"))
[facebook.net] HTTPConnectionPool(host='facebook.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281dc1bd10>: Failed to resolve 'facebook.net' ([Errno -5] No address associated with hostname)"))
[ggpht.com] HTTPConnectionPool(host='ggpht.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d28d110>: Failed to resolve 'ggpht.com' ([Errno -5] No address associated with hostname)"))
[googletagservices.com] HTTPConnectionPool(host='googletagservices.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e29f650>: Failed to resolve 'googletagservices.com' ([Errno -5] No address associated with hostname)"))
[googleusercontent.com] HTTPConnectionPool(host='googleusercontent.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d8f6010>: Failed to resolve 'googleusercontent.com' ([Errno -5] No address associated with hostname)"))
[gtld-servers.net] HTTPConnectionPool(host='gtld-servers.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281da6cd50>: Failed to resolve 'gtld-servers.net' ([Errno -5] No address associated with hostname)"))
[gvt1.com] HTTPConnectionPool(host='gvt1.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818447410>: Failed to resolve 'gvt1.com' ([Errno -5] No address associated with hostname)"))
[gvt2.com] HTTPConnectionPool(host='gvt2.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f2818444ed0>: Failed to resolve 'gvt2.com' ([Errno -5] No address associated with hostname)"))
[health.mil] ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[l-msedge.net] HTTPConnectionPool(host='l-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281dc38610>: Failed to resolve 'l-msedge.net' ([Errno -5] No address associated with hostname)"))
[macromedia.com] HTTPConnectionPool(host='macromedia.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f281e057f10>, 'Connection to macromedia.com timed out. (connect timeout=None)'))
[mangosip.ru] HTTPConnectionPool(host='mangosip.ru', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f281853e5d0>, 'Connection to mangosip.ru timed out. (connect timeout=None)'))
[microsoftonline.com] HTTPConnectionPool(host='microsoftonline.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281df62150>: Failed to resolve 'microsoftonline.com' ([Errno -5] No address associated with hostname)"))
[miit.gov.cn] HTTPConnectionPool(host='miit.gov.cn', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281df3e050>: Failed to resolve 'miit.gov.cn' ([Errno -5] No address associated with hostname)"))
[msedge.net] HTTPConnectionPool(host='msedge.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f281db7c710>, 'Connection to msedge.net timed out. (connect timeout=None)'))
[msftncsi.com] HTTPConnectionPool(host='msftncsi.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281db7fed0>: Failed to resolve 'msftncsi.com' ([Errno -5] No address associated with hostname)"))
[netflix.net] HTTPConnectionPool(host='netflix.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f281dafa810>, 'Connection to netflix.net timed out. (connect timeout=None)'))
[nflxso.net] HTTPConnectionPool(host='nflxso.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281db8c750>: Failed to resolve 'nflxso.net' ([Errno -5] No address associated with hostname)"))
[nih.gov] ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[nr-data.net] HTTPConnectionPool(host='nr-data.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281dd4ee50>: Failed to resolve 'nr-data.net' ([Errno -5] No address associated with hostname)"))
[o365filtering.com] HTTPConnectionPool(host='o365filtering.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e2ad290>: Failed to resolve 'o365filtering.com' ([Errno -5] No address associated with hostname)"))
[office.net] HTTPConnectionPool(host='office.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281df367d0>: Failed to resolve 'office.net' ([Errno -5] No address associated with hostname)"))
[omtrdc.net] HTTPConnectionPool(host='omtrdc.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e631b90>: Failed to resolve 'omtrdc.net' ([Errno -5] No address associated with hostname)"))
[openx.net] HTTPConnectionPool(host='openx.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d114c90>: Failed to resolve 'openx.net' ([Errno -5] No address associated with hostname)"))
[rbxcdn.com] HTTPConnectionPool(host='rbxcdn.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281855a710>: Failed to resolve 'rbxcdn.com' ([Errno -5] No address associated with hostname)"))
[registrar-servers.com] HTTPConnectionPool(host='registrar-servers.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281df9ef90>: Failed to resolve 'registrar-servers.com' ([Errno -5] No address associated with hostname)"))
[rocket-cdn.com] HTTPConnectionPool(host='rocket-cdn.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281df086d0>: Failed to resolve 'rocket-cdn.com' ([Errno -5] No address associated with hostname)"))
[root-servers.net] HTTPConnectionPool(host='root-servers.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e088410>: Failed to resolve 'root-servers.net' ([Errno -5] No address associated with hostname)"))
[rubiconproject.com] ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
[ruidongcloud.com] HTTPConnectionPool(host='ruidongcloud.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e02bed0>: Failed to resolve 'ruidongcloud.com' ([Errno -5] No address associated with hostname)"))
[s-msedge.net] HTTPConnectionPool(host='s-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e029f50>: Failed to resolve 's-msedge.net' ([Errno -5] No address associated with hostname)"))
[spo-msedge.net] HTTPConnectionPool(host='spo-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281dbd6850>: Failed to resolve 'spo-msedge.net' ([Errno -5] No address associated with hostname)"))
[t-msedge.net] HTTPConnectionPool(host='t-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e2239d0>: Failed to resolve 't-msedge.net' ([Errno -5] No address associated with hostname)"))
[tiktokcdn.com] HTTPConnectionPool(host='tiktokcdn.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281de1c5d0>: Failed to resolve 'tiktokcdn.com' ([Errno -5] No address associated with hostname)"))
[trafficmanager.net] HTTPConnectionPool(host='trafficmanager.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f28185a40d0>: Failed to resolve 'trafficmanager.net' ([Errno -5] No address associated with hostname)"))
[trbcdn.net] ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
[ttlivecdn.com] HTTPConnectionPool(host='ttlivecdn.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281de13110>: Failed to resolve 'ttlivecdn.com' ([Errno -5] No address associated with hostname)"))
[userapi.com] ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
[wac-msedge.net] HTTPConnectionPool(host='wac-msedge.net', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e1f9990>: Failed to resolve 'wac-msedge.net' ([Errno -5] No address associated with hostname)"))
[windowsupdate.com] HTTPConnectionPool(host='windowsupdate.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281d918250>: Failed to resolve 'windowsupdate.com' ([Errno -5] No address associated with hostname)"))
[wsdvs.com] HTTPConnectionPool(host='wsdvs.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281dbe8ed0>: Failed to resolve 'wsdvs.com' ([Errno -5] No address associated with hostname)"))
[ytimg.com] HTTPConnectionPool(host='ytimg.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x7f281e0c49d0>: Failed to resolve 'ytimg.com' ([Errno -5] No address associated with hostname)"))
